pkg/kn/config/config.go

```diff
@@ -159,12 +190,15 @@ func BootstrapConfig() error {
 // Add bootstrap flags use in a separate bootstrap proceeds
 func AddBootstrapFlags(flags *flag.FlagSet) {
 	flags.StringVar(&globalConfig.configFile, "config", "", fmt.Sprintf("kn configuration file (default: %s)", defaultConfigFileForUsageMessage()))
+	flags.BoolVar(&globalConfig.skipInitCfg, "no-init", false, "Skip initialize configuration file")
+	flags.BoolVar(&globalConfig.reInitCfg, "init", false, "Force initialization to configuration file")
```

Comment:
I'd vote for something more descriptive though a bit longer, e.g `--init-config`, 

On the other hand I'm not entirely sure that those flags are needed, even though original issues has proposed them. :)
/cc @rhuss